clang/include/clang/Sema/Sema.h

```diff
@@ -13125,8 +13125,8 @@ class Sema final {
                                    SourceLocation BuiltinLoc,
                                    SourceLocation RParenLoc);
 
-  bool isSyclGlobalVariableAllowedType(QualType Ty);
-  bool isSyclDeviceGlobalType(QualType Ty);
+  template <typename T>
+  bool isSyclGlobalType(QualType Ty);
```

Comment:
Since this is a template now, you need to define it in the header, othewise, I suppose, it won't instantiate correctly if defined in one .cpp file and used from another.

Reply:
Indeed. Moved the definition to Sema.h header.

Reply:
https://github.com/intel/llvm/pull/5597/commits/679b5f02a36576b026770098fc602cb234c6964d